pkg/commands/generate_new_bundle_test.go

```diff
@@ -10,23 +11,20 @@ import (
 	"github.com/massdriver-cloud/mass/pkg/commands"
 	"github.com/massdriver-cloud/mass/pkg/mockfilesystem"
 	"github.com/massdriver-cloud/mass/pkg/templatecache"
-	"github.com/spf13/afero"
 	"sigs.k8s.io/yaml"
 )
 
 func TestCopyFilesFromTemplateToCurrentDirectory(t *testing.T) {
-	rootTemplateDir := "/home/md-cloud"
-	writePath := "."
+	testDir := t.TempDir()
+	rootTemplateDir := path.Join(testDir, "/home/md-cloud")
+	writePath := testDir
```

Comment:
would look nicer to remove L20 and replace all `writePath` to `testDir`, or replace `testDir` with `writePath`